chapters/synchronous.tex

```diff
@@ -449,7 +449,7 @@ \section{Clock Constructors}\doublelabel{clock-constructors}
 
 The input argument, \lstinline!interval!, is a clocked Component Expression (see
 \autoref{argument-restrictions-component-expression}) or a parameter expression.
-The \lstinline!interval! must be strictly positive (\lstinline!interval>0.0!) of type \textbf{Real} unit ``s''.
+The \lstinline!interval! must be strictly positive (\lstinline!interval>0.0!) of type \textbf{Real} unit \emph{s}.
```

Comment:
And similarly of type \lstinline!Real(unit="s")!. (Or some variation of this.)

Reply:
Fixed.